src/back/glsl/mod.rs

```diff
@@ -2005,11 +2013,27 @@ impl<'a, W: Write> Writer<'a, W> {
                                         if let Some(crate::Binding::BuiltIn(builtin)) =
                                             member.binding
                                         {
+                                            has_point_size |= builtin == crate::BuiltIn::PointSize;
+
                                             match builtin {
                                                 crate::BuiltIn::ClipDistance
-                                                | crate::BuiltIn::CullDistance
-                                                | crate::BuiltIn::PointSize => {
+                                                | crate::BuiltIn::CullDistance => {
                                                     if self.options.version.is_es() {
+                                                        // According to https://github.com/KhronosGroup/GLSL/issues/132
+                                                        /* > We agreed that gl_ClipDistance and gl_CullDistance, and related language,
+                                                            should not appear in the ESSL spec since it's not part of OpenGL ES 3.2.
+                                                        */
+                                                        if let Version::Embedded {
+                                                            version,
+                                                            is_webgl: _,
+                                                        } = self.options.version
+                                                        {
+                                                            if version <= 320 {
+                                                                log::warn!(
+                                                            "{:?} is not part of OpenGL ES <= 3.2",
+                                                            builtin);
+                                                            }
+                                                        }
```

Comment:
I think we should log the warning regardless of version since we found out that no gles version supports those.